examples/basic.js

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
I found this file confusing because of the usage of `SyncClient` - based on the naming, I expected to receive a client that had only synchronous methods, but on further investigation, the constructor is synchronous and the methods hanging off of it are async.

I'm not sure which is more intuitive.

Reply:
It's not you, it's me :)  addressed in https://github.com/godaddy/kubernetes-client/pull/225